.github/workflows/omni.yml

```diff
@@ -59,29 +59,26 @@ jobs:
         const DEFAULT_TEST_GROUP = process.env.MATRIX_DEFAULT_TEST_GROUP || 'fast';
 
         const JDK_OPTIONS = [
-            { group: 'Zulu', version: '8', lts: true, distribution: 'zulu' },
-            { group: 'Zulu', version: '11', lts: true, distribution: 'zulu' },
-            { group: 'Zulu', version: '15', lts: false, distribution: 'zulu' },
+            { group: 'Zulu', version: '17', lts: true, distribution: 'zulu' },
+            { group: 'Zulu', version: '21', lts: true, distribution: 'zulu' },
         ];
 
         const OTHER_JDK_OPTIONS = [
           // Adopt
-          { group: 'Adopt Hotspot', version: '8', lts: true, distribution: 'adopt-hotspot' },
-          { group: 'Adopt Hotspot', version: '11', lts: true, distribution: 'adopt-hotspot' },
+          { group: 'Adopt Hotspot', version: '17', lts: true, distribution: 'adopt-hotspot' },
+          { group: 'Adopt Hotspot', version: '21', lts: true, distribution: 'adopt-hotspot' },
 
           // Adopt OpenJ9
           // TODO: Replace these hard coded versions with something that dynamically picks the most recent
-          { group: 'Adopt OpenJ9', version: '8', lts: true, distribution: 'adopt-openj9'},
-          { group: 'Adopt OpenJ9', version: '11', lts: true, distribution: 'adopt-openj9'},
+          { group: 'Adopt OpenJ9', version: '17', lts: true, distribution: 'adopt-openj9'},
+          { group: 'Adopt OpenJ9', version: '21', lts: true, distribution: 'adopt-openj9'},
 
           // Amazon Corretto
-          { group: 'Corretto', version: '8', lts: true, distribution: 'jdkfile', url: 'https://corretto.aws/downloads/latest/amazon-corretto-8-x64-linux-jdk.tar.gz'},
-          { group: 'Corretto', version: '11', lts: true, distribution: 'jdkfile', url: 'https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz'},
+          { group: 'Corretto', version: '17', lts: true, distribution: 'jdkfile', url: 'https://corretto.aws/downloads/latest/amazon-corretto-17-x64-linux-jdk.tar.gz'},
+          { group: 'Corretto', version: '21', lts: true, distribution: 'jdkfile', url: 'https://corretto.aws/downloads/latest/amazon-corretto-21-x64-linux-jdk.tar.gz'},
         ];
 
         const PG_VERSIONS = [
-            '8.4',
```

Comment:
just a heads up, the web page mentiones 9.1 as minimum tested.

Reply:
looking at https://github.com/pgjdbc/pgjdbc/blob/032d0e225a91e866d7dae680ebb784507392e803/.github/workflows/omni.yml#L81 that is correct.